assets/css/style.scss

```diff
@@ -143,7 +143,6 @@
 			left: 0;
 			right: 0;
 			position: absolute;
-			color: #000;
```

Comment:
Removing this line changes the color of the stars in product grid blocks.

All Products block without styles applied:

| Before | After |
| --- | --- |
| ![imatge](https://user-images.githubusercontent.com/3616980/152954364-991d0912-4c91-4ebc-a513-c69dd5698f56.png) | ![imatge](https://user-images.githubusercontent.com/3616980/152954407-1e618722-2958-4999-bbcd-faf398eb2b8e.png) |

Top Rated Products block:

| Before | After |
| --- | --- |
| ![imatge](https://user-images.githubusercontent.com/3616980/152954512-530bdf45-d606-4112-bba3-ec35647083c9.png) | ![imatge](https://user-images.githubusercontent.com/3616980/152954567-440086c0-8a9e-4916-8bec-c9218071c014.png) |

Do you think there is any way to keep this declaration while getting global styles to work?

Reply:
Thanks for catching this!

I restored the previous CSS with [84f00de](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5521/commits/84f00de3c75c66f81b100c36d0890ca3f62b7a92) commit.

I forgot that we have some components that display product ratings on the PHP side (as ´Top Rated Products` block). Now the global style will be applied on PHP side blocks, (but as default the color will be black)

I'm not sure about the screenshot that you shared. 
![image](https://user-images.githubusercontent.com/4463174/153015885-b20653c3-b860-494b-9a52-bbc5d8825817.png)

I checked the trunk and I see always black stars. Can you check again?

Reply:
Ah, not sure what I did to get that color. :sweat_smile: It was in Storefront but I checked it again and it's dark-gray as the other blocks.